src/field/MagField.hh

```diff
@@ -18,27 +18,20 @@ namespace celeritas
  */
 class MagField
 {
-  public:
-    //@{
-    //! Type aliases
-    using field_value = Real3;
-    using field_type  = bool;
-    //@}
-
   public:
     //! Construct from a uniform field
-    CELER_FUNCTION MagField(const field_value& value);
+    CELER_FUNCTION MagField(const Real3& value);
 
     //! Return a magnetic field value at a given position
-    CELER_FUNCTION auto operator()(const Real3& position) const -> field_value;
+    CELER_FUNCTION Real3 operator()(const Real3& position) const;
 
     //! Interface for a position-dependent magnetic field
-    CELER_FUNCTION auto get_field(const Real3& position) const -> field_value;
+    CELER_FUNCTION Real3 get_field(const Real3& position) const;
```

Comment:
Not sure this function is necessary? Shouldn't the field just need the `operator()`? Does it also need to be overloaded on `OdeState`?

Reply:
If you remember, you wanted to add a place-holder for a position dependent operator.  I removed the position dependent extension until it is actually needed.